tests/drivers/disk/disk_access/boards/qemu_x86_64.overlay

```diff
@@ -4,11 +4,6 @@
 
 / {
 	pcie0 {
-		#address-cells = <1>;
```

Comment:
this patch is fine, but should come in the history **before** the patch that remove "intel,pcie"

Reply:
updated